lib/command_info.rb

```diff
@@ -24,7 +24,6 @@ def order
     "To place an order | `order: food`\n" \
     "To see someone elses order | `order? @name_of_person`\n" \
     "To see all orders | `all orders?`\n" \
-    "To see all orders with user ids | `everyone`\n" \
```

Comment:
Arg, I messed up. I should have done that on each PR deleting a command :octocat: